Lib/xml/dom/minidom.py

```diff
@@ -80,10 +82,16 @@ def _get_lastChild(self):
             return self.childNodes[-1]
 
     def insertBefore(self, newChild, refChild):
+        """Insert a new DOM Node before an existing Node.
```

Comment:
Blank line is needed.
```suggestion
        """Insert a new DOM Node before an existing Node.

```
I think first url should be just above 2nd.